src/Uno.UI.Tests/Windows_UI_Xaml_Data/xBindTests/Given_xBind_Functions.cs

```diff
@@ -11,24 +11,24 @@ public void When_Initial_Value()
 		{
 			var SUT = new Functions_Control();
 
-			Assert.IsNull(SUT._InstanceProperty.Text);
-			Assert.IsNull(SUT._StaticProperty.Text);
-			Assert.IsNull(SUT._StaticPrivateProperty.Text);
-			Assert.IsNull(SUT._StaticPrivateReadonlyField.Text);
-			Assert.IsNull(SUT._InstanceDP.Text);
-			Assert.IsNull(SUT._InnerProperty.Text);
-			Assert.IsNull(SUT._StaticClass_PublicStaticProperty.Text);
-			Assert.IsNull(SUT._InstanceFunction_OneParam.Text);
-			Assert.IsNull(SUT._InstanceFunction_OneParam_DP_Update_OneTime.Text);
-			Assert.IsNull(SUT._InstanceFunction_OneParam_DP_Update_OneWay.Text);
-			Assert.IsNull(SUT._InstanceFunction_TwoParam.Text);
-			Assert.IsNull(SUT._InstanceFunction_TwoParam_WithConstant.Text);
-			Assert.IsNull(SUT._System_Function.Text);
-			Assert.IsNull(SUT._System_Function_with_Quote.Text);
-			Assert.IsNull(SUT._InstanceFunction_Parameterless.Text);
-			Assert.IsNull(SUT._StaticFunction_Parameterless.Text);
-			Assert.IsNull(SUT._InstanceFunction_Boolean_False.Text);
-			Assert.IsNull(SUT._InstanceFunction_Boolean_True.Text);
+			Assert.AreEqual(string.Empty, SUT._InstanceProperty.Text);
```

Comment:
Hmm funny story...

I had to change the tests because I switched `NET461` to use the main `TextBlock.cs` instead of a dummy (I forget why). On Android/iOS/WASM the default is (and probably always was) `string.Empty`: https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.cs#L211

I checked just now and UWP gives two different answers:
``` csharp
			var textDefault = TextBlock.TextProperty.GetMetadata(typeof(TextBlock)).DefaultValue as string; // null
			var textDefault2 = (new TextBlock()).Text; // ""
```

So I think the correct value is `string.Empty`, and the DP default on UWP is a lie (as for `FontSize`).